config/deploy.rb

```diff
@@ -21,7 +21,7 @@
 # set :rbenv_ruby, File.read('.ruby-version').strip
 
 set :rbenv_prefix, "RBENV_ROOT=#{fetch(:rbenv_path)} RBENV_VERSION=#{fetch(:rbenv_ruby)} #{fetch(:rbenv_path)}/bin/rbenv exec"
-set :rbenv_map_bins, %w{rake gem bundle ruby rails}
+# set :rbenv_map_bins, %w{rake gem bundle ruby rails}
```

Comment:
@Julianobsg ?

Reply:
This line was removed because we needed to add sidekiq and sidekiq_cti to map bins, but I saw a recommendation to remove this line altogether in the version that we use.

Reply:
So this is not going to hurt us? Then just remove the line.

Reply:
It's not needed and recommended to remove, so we remove it.